tests/interpreters/test_riscv_scf_interpreter.py

```diff
@@ -43,6 +43,50 @@ def for_loop_region(args: tuple[BlockArgument, ...]):
         func.Return(result)
 
 
+def sum_to_while_fn(n: int) -> int:
+    """
+    Python implementation of sum_to_while_op
+    """
+    result = 0
+    i = 0
+    while i < n:
+        result += 1
+        i += 1
+    return result
```

Comment:
Should we comment this if this isn't called?

Reply:
Ah yeah I mean it's basically documentation becvause python is more readable than IR with builders, I'll comment it out.